modules/log/logger_global.go

```diff
@@ -57,11 +57,13 @@ func Critical(format string, v ...any) {
 	Log(1, ERROR, format, v...)
 }
 
+var OsExiter = os.Exit
+
```

Comment:
What's the point of this cache? Also I would prefer it to be named `OsExit` if it stays.

Reply:
I do not understand what's the point of the question. It doesn't work without this assignment.

Reply:
What does this line do? As I see it, it stores a reference to the function in a "global" variable so should work without?

Reply:
Ah I see you call it in other files. Still I find it strange that we don't use `os.Exit()` directly. It seems like a useless abstraction.

Reply:
It is useful. For this case, it collects the stacktrace. For the future, it could test code with `log.Fatal` without really exiting.

Reply:
Ok. BTW I would like to call it `log.OsExit` but I see the current name is consistent with `cli` package.